sbt-plugin/src/sbt-test/scala3/basic/build.sbt

```diff
@@ -0,0 +1,8 @@
+enablePlugins(ScalaJSPlugin)
+
+scalaVersion := "3.0.0-M1"
+
+// Test withDottyCompat for %%% dependencies
+libraryDependencies += ("org.scala-js" %%% "scalajs-dom" % "1.1.0").withDottyCompat(scalaVersion.value)
```

Comment:
Can we depend on something else here (maybe a dummy library if `withDottyCompat` needs to be tested here)? I'm just a bit concerned about the dependency this introduces in the library graph. (it would be an actual problem if Scala.js were to publish a new major version for example).

Reply:
I switch to depending on scalajs-ir.